pkg/apis/consts/consts.go

```diff
@@ -0,0 +1,31 @@
+/*
+Portions Copyright (c) Microsoft Corporation.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package consts
+
+const (
+	NetworkPluginAzure   = "azure"
+	NetworkPluginKubenet = "kubenet"
+
+	PodNetworkTypeOverlay = "overlay"
+	PodNetworkTypeNone    = ""
```

Comment:
There are 3 pod network types but I am only using two in the code right now so just adding these two. 

But really there is also a PodNetworkType for pod subnet as well.